goodfood/complain/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+# Create your models here.
+
+class Post(models.Model):
```

Comment:
Sorry for not elaborating in the issue;
but a better model name would be `Complain` and form will be `ComplainForm`.